### PR TITLE
docs: clarify microsoft.md OIDC setup

### PR DESCRIPTION
**Pull request by AlexanderSarson**
It was unclear the url was needed to be specified.

Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [ ] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [ ] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] I've included "Closes [ISSUE #]" or "Fixes [ISSUE #]" in the description to automatically close the associated issue.
* [ ] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [ ] Optional. My organization is added to USERS.md.
* [ ] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo)
* [ ] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [ ] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/latest/developer-guide/ci/)).

### CODE REVIEW

File: docs/operator-manual/user-management/microsoft.md [hunk 1]
```diff
@@ -105,12 +105,12 @@ policy.default: role:readonly
...
     ![Azure AD token configuration](../../assets/azure-token-configuration.png "Azure AD token configuration")
 
-3. Edit `argocd-cm` and configure the `data.oidc.config` section:
+3. Edit `argocd-cm` and configure the `data.oidc.config` and `data.url` section:
 
         ConfigMap -> argocd-cm
 
         data:
-            url: https://argocd.example.com/
+            url: {argocd_externally_facing_base_url}  # https://argocd.example.com/
```
Comment (jannfis) [inline docs/operator-manual/user-management/microsoft.md:113]:
```suggestion
            url: https://argocd.example.com/  # Replace with the external base URL of your Argo CD
```

**Review by jannfis — CHANGES_REQUESTED** [review]
Thanks!

I have a minor request for change, please refer to the suggestion.

**Comment by pasha-codefresh** [comment]
@AlexanderSarson could you please merge conflicts ?

**Review by jannfis — APPROVED** [review]
LGTM

**Pull request merged**